strr-examiner-web/app/components/Host/SupportingInfo.vue

```diff
@@ -4,18 +4,18 @@ import { ConnectPageSection } from '#components'
 import { useExaminerStore } from '~/stores/examiner'
 import { useFlags } from '~/composables/useFlags'
 
-const props = defineProps<{
-  application: HostApplicationResp | undefined
-}>()
-const reg = props.application?.registration
-
+const { isApplication, activeRecord } = useExaminerStore()
```

Comment:
To maintain reactivity, it is better to set these as refs, in case the store gets updated (with new values), it needs to be updated here as well. Use `storeToRefs()`.

Reply:
Done